src/ResourceManager/KeyVault/Commands.KeyVault/help/Remove-AzureRmKeyVaultNetworkRuleSet.md

```diff
@@ -0,0 +1,219 @@
+---
+external help file: Microsoft.Azure.Commands.KeyVault.dll-Help.xml
+Module Name: AzureRM.KeyVault
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.keyvault/add-azurermkeyvaultnetworkruleset
+schema: 2.0.0
+---
+
+# Remove-AzureRmKeyVaultNetworkRuleSet
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+### ByVaultName (Default)
+```
+Remove-AzureRmKeyVaultNetworkRuleSet [-VaultName] <String> [[-ResourceGroupName] <String>]
+ [-IpAddressRange <String[]>] [-VirtualNetworkResourceId <String[]>] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByInputObject
+```
+Remove-AzureRmKeyVaultNetworkRuleSet [-InputObject] <PSKeyVault> [-IpAddressRange <String[]>]
+ [-VirtualNetworkResourceId <String[]>] [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Remove-AzureRmKeyVaultNetworkRuleSet [-ResourceId] <String> [-IpAddressRange <String[]>]
+ [-VirtualNetworkResourceId <String[]>] [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> PS C:\> Remove-AzureRmKeyVaultNetworkRuleSet -VaultName myVault -IpAddressRange "10.0.0.1" -VirtualNetworkResourceId $myNetworkResId 
```

Comment:
Same comment here - add a cmdlet above for obtaining the VirtualNetwowrkResourceId

Reply:
The IP address is in CIDR format such as 1.0.0.0/26 and 1.0.0.0/28